main.py

```diff
@@ -6,17 +6,18 @@
 
 def main():
     # import data
-    train_spectrograms = load_dump(config.TRAIN_DATA_PATH)
+    train_spectrograms = load_dump(config.DEV_DATA_PATH if config.IS_DEV_MODE else config.TRAIN_DATA_PATH)
 
     # import labels
-    train_labels = load_dump(config.TRAIN_LABELS_PATH)
+    train_labels = load_dump(config.DEV_LABELS_PATH if config.IS_DEV_MODE else config.TRAIN_LABELS_PATH)
 
     # generate dataset
     def generate_subspectrogram():
         for i in range(len(train_labels)):
-            sub_spectro = draw_subspectrogram(train_spectrograms[i], 5)
+            sub_spectro = draw_subspectrogram(train_spectrograms[i])
             tensor_spectro = tf.convert_to_tensor(sub_spectro)
-            tensor_spectro = tf.reshape(tensor_spectro, (128, 430, 1))
+            tensor_spectro = tf.transpose(tensor_spectro)
+            tensor_spectro = tf.reshape(tensor_spectro, (config.SUBSPECTROGRAM_POINTS, config.MEL_POINTS, 1))
```

Comment:
We could move this reshape step to the convolution model, as it is not needed for the GRU model